src/specs/models/samples-bodies/Samples.v2.yaml

```diff
@@ -0,0 +1,4 @@
+title: Sample object
+description: Schema for a sample object
+items:
+  $ref: ./Sample.v2.yaml
```

Comment:
This object looks pretty simple, we could just move it to the main schema api.v2.yaml perhaps (or move it into Sample.v2.yaml with the rename to `GetSamples.v2.yaml`)